bindings/python/pydeck/docs/data_utils.md

```diff
@@ -0,0 +1,43 @@
+pydeck.data\_utils
+=======
+
+## pydeck.data\_utils.autocompute\_viewport
+
+```python
+def autocompute_viewport(
+    points,
+    view_proportion=1,
+    viewport_type=pydeck.ViewState)
+```
+
+Automatically computes a zoom level for the points passed in.
+
+#### Parameters
+`points` : `list` of `list` of `float` or `pandas.core.frame`
+    A list of points
+`view_propotion` : `float`
+    Proportion of the data that is meaningful to plot
```

Comment:
I don't understand what this param does based on this description, but maybe that is OK...